chart/skywalking/values.yaml

```diff
@@ -41,6 +41,7 @@ oap:
     # zabbix: 10051
     grpc: 11800
     rest: 12800
+    zipkinUI: 9412
```

Comment:
I think this should not be enabled by default?

Reply:
Ok. Maybe I should `if` to create a conditional block in `ui-deployment.yaml`.